README.md

```diff
@@ -7,9 +7,20 @@ Extract and copy the contents from Android Archive (.aar) files.
 ## Usage
 This module is used to extract the contents of an Android Archive (.aar) to a desired location and optionally copy any bundled assets and libraries to a new location.
 
+> Note: All paths are expected to be absolute (you can use `path.join()` for that).
+
 ```js
-var AarTransformer = require('appc-aar-transform');
-var transformer = new AarTransformer(logger) // logger is optional, can be any bunyan based logger
+var AARTransformer = require('appc-aar-transform');
+var transformer = new AARTransformer(logger) // logger is optional, can be any bunyan based logger
+
+var options = {
+  aarPathAndFilename: 'path/to/file.aar',
```

Comment:
Please add a leading slash so its obvious this and the following options are expected to be absolute paths.